orchagent/notifications.cpp

```diff
@@ -10,6 +12,38 @@ extern "C" {
 
 extern mutex gDbMutex;
 extern PortsOrch *gPortsOrch;
+extern FdbOrch *gFdbOrch;
+
+void on_fdb_event(uint32_t count, sai_fdb_event_notification_data_t *data)
+{
+    SWSS_LOG_ENTER();
+
+    lock_guard<mutex> lock(gDbMutex);
+
+    if (!gFdbOrch)
+    {
+        SWSS_LOG_NOTICE("gFdbOrch is not initialized");
+        return;
+    }
+
+    for (uint32_t i = 0; i < count; ++i)
+    {
+        sai_object_id_t oid = SAI_NULL_OBJECT_ID;
+
+        for (uint32_t j = 0; j < data[i].attr_count; ++j)
+        {
+            if (data[i].attr[j].id == SAI_FDB_ENTRY_ATTR_PORT_ID)
+            {
+                oid = data[i].attr[j].value.oid;
+                break;
+            }
+        }
+
+        assert(oid != SAI_NULL_OBJECT_ID);
```

Comment:
If used Mellanox SAI directly such code will fail. With SAI Redis it doesn't fail because ASIC DB contains information about port and sends event with port ID attribute. Removed assert to follow general concept